tools/Build.flow

```diff
@@ -149,20 +149,6 @@ Using --flags:test will include all source and configuration flagged as 'test' t
   <li></li>
 </ul>
 
-<h1>Additional Tooling</h1>
```

Comment:
Why remove this?   This is a build feature and this flow documentation for the build (albeit incomplete). 
Otherwise, Ok with removing Recipe if you don't want it.